apps/mitt-konto/src/MittKonto/Main/Helpers.purs

```diff
@@ -0,0 +1,62 @@
+module MittKonto.Main.Helpers where
+
+import Prelude
+
+import Data.Array ((:))
+import Data.Foldable (oneOf)
+import Data.JSDate (JSDate, parse)
+import Data.Maybe (Maybe(..))
+import Effect (Effect)
+import Effect.Aff (Aff)
+import Effect.Aff as Aff
+import Effect.Class (liftEffect)
+import Effect.Exception (Error, error)
+import Effect.Unsafe (unsafePerformEffect)
+import KSF.Alert.Component (Alert)
+import KSF.Alert.Component as Alert
+import KSF.Error as KSF.Error
+import KSF.Spinner as Spinner
+import KSF.User (SubscriptionPayments, User)
+import MittKonto.Main.Types as Types
+import React.Basic (JSX)
+import React.Basic.DOM as DOM
+
+setLoading :: Maybe Spinner.Loading -> Types.State -> Types.State
```

Comment:
I feel these would fit better within the same file where the type they're manipulating is located in. They seem pretty lonely in here and checking on what they do would require opening that other file as well.

Reply:
Most of them are not used in a single file anymore and I'm strongly against duplication. Moreover I think that their names are self explanatory, I think I would only check them if I had to change them and that's a big part of the point of this refactoring to me.

Thoughts?

Reply:
I meant moving them to the same file where State is defined, not copying them.

Reply:
Aaah so in the `Types.purs` file! Yes that is a very good idea, sorry I really misunderstood :smile: 

So you mean the `setSomething` functions right?